src/OmnichannelChatSDK.ts

```diff
@@ -712,7 +821,10 @@ class OmnichannelChatSDK {
                     SpoolSDK: spoolSDKCDNUrl
                 });
 
-                this.scenarioMarker.startScenario(TelemetryEvent.GetVoiceVideoCalling);
+                this.scenarioMarker.startScenario(TelemetryEvent.GetVoiceVideoCalling, {            
```

Comment:
Please remove `RequestId` & `ChatId` since they may not be relevant at this point of time.